download.html

```diff
@@ -285,29 +285,72 @@ <h1>Apache Ignite <strong>Downloads</strong></h1>
+                            <ul>
+                                <li><a href="#sources">Source Releases</a></li>
+                                <li><a href="#binaries">Binary Releases</a></li>
+                                <li><a href="#build-source">Building the Binaries</a></li>
+                                <li><a href="#docker">Docker and Cloud Images</a></li>
+                                <li><a href="#git-repository">Git Repository</a></li>
+                            </ul>
+                        </div>
+                        <div class="col-sm-6">
+                            <ul>
+                                <li><a href="#maven">Download from Maven</a></li>
+                                <li><a href="#nuget">Download from NuGet</a></li>
+                                <li><a href="#deb-package">DEB Package Installation</a></li>
+                                <li><a href="#rpm-package">RPM Package Installation</a></li>
+                                <li><a href="#3rdparty">3<sup>rd</sup> Party Binaries</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <p>
+                        To get LGPL dependencies (e.g. Hibernate L2 Cache),
+                        see <a href="#build-source">Building The Binaries</a>.</p>
                 </div>
             </div>
-            <p>* To get LGPL dependencies (e.g. Hibernate L2 Cache), see <a href="#build-source">Building The
-                Binaries</a>.</p>
+            <p>
+                If you encounter a problem with the selected mirror, please select another mirror.
+                If all mirrors are failing, there are <i>backup</i> mirrors
+                (at the end of the mirrors list) that should be available.</p>
+            <form action="/download.cgi" method="get" id="SelectMirror">
+                Selected mirror:
+                <select name="Preferred" onchange="submit()">
+                    [if-any http]
+                    [for http]
+                    <option value="[http]" [is http preferred] selected="selected" [end]>[http]</option>
+                    [end]
+                    [end]
+                    [if-any ftp]
+                    [for ftp]
+                    <option value="[ftp]" [is ftp preferred] selected="selected" [end]>[ftp]</option>
+                    [end]
+                    [end]
+                    [if-any backup]
+                    [for backup]
+                    <option value="[backup]" [is backup preferred] selected="selected" [end]>[backup] (backup)</option>
+                    [end]
+                    [end]
+                </select>
+                <input type="submit" value="Change"/>
+            </form>
         </section>
 
         <section id="sources" class="page-section">
             <h2>Source Releases</h2>
+            <p>
+                Binary downloads are provided only for the convenience of our users
```

Comment:
Please change the verbatim to the one below. Also, why do we place the paragraph under "Source Releases" section?

_Binary release packages are provided for your convenience and not considered as primary release artifacts of the ASF. It's recommended to verify a release downloadable, following <a href="#verification">this guidelines</a>._

Reply:
Fixed